.github/workflows/performance.yml

```diff
@@ -0,0 +1,62 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'feature/**'
```

Comment:
curious why feature branches are included here, I thought we were only checking trunkmerges?

Reply:
Good catch. @felixarntz had proposed this change in all the core workflows in [Trac #57556](https://core.trac.wordpress.org/ticket/57556) so we should probably change to whatever the resolution of that ticket is eventually and only log data when a commit is merged to `trunk`.

Reply:
@adamsilverstein I believe it's because we're working in a feature branch right now. Once we would open this as a PR to core, we may or may not remove this. See https://core.trac.wordpress.org/ticket/57556 for context.

Reply:
@joemcgill You beat me to it, didn't see your comment yet when I wrote mine :D